app/src/main/java/com/mapbox/maps/testapp/examples/LocationComponentActivity.kt

```diff
@@ -93,21 +72,6 @@ class LocationComponentActivity : AppCompatActivity() {
     }
   }
 
-  private fun setupBuildings(style: Style) {
-    val fillExtrusionLayer = fillExtrusionLayer("3d-buildings", "composite") {
```

Comment:
Just curious: we could keep 3d landmarks, or we are removing that too with model-layer ?